contracts/staking/RewardDelegators.sol

```diff
@@ -524,4 +524,21 @@ contract RewardDelegators is
             return 0;
         }
     }
+    
+    // ------- receiver payments ------------------ //
+
+    event AddReceiverBalance(address indexed receiver, uint256 amount);
+    event UpdateReceiverRewardPerEpoch(address indexed receiver, uint256 amount);
+
+    function addReceiverBalance(address receiver, uint256 amount) public {
+        require(receiver != address(0), "CRW: address 0");
```

Comment:
Add one more sanity check on  amount != 0

Reply:
format for naming of  error messages is incorrect.

Reply:
Changed